webapp/client/src/lib/translations.js

```diff
@@ -282,12 +282,13 @@ const translations = {
       heading: "Wie geht es weiter?",
       text: "Ihre Steuererklärung wird von Ihrem Finanzamt bearbeitet. Sie bekommen Ihren Steuerbescheid per Post zugeschickt.",
     },
+    recommend: {
+      heading: "Empfehlen Sie uns weiter!",
+      text: "Wir möchten noch mehr Menschen bei Ihrer Steuererklärung helfen! Sie können uns dabei helfen, indem Sie den Steuerlotsen bekannter machen. Empfehlen Sie uns Ihren Bekannten und Freunden!",
+    },
     logout: {
       heading: "Wir löschen Ihr Nutzerkonto",
-      text1:
-        "Bitte beachten Sie, dass Sie Ihren Freischaltcode nicht erneut verwenden können. Wenn Sie unseren Service nächstes Jahr wieder nutzen möchten, können Sie sich einfach erneut registrieren.",
-      text2:
-        "Sie haben sich Ihr Transferticket notiert und Ihre Übersicht der übermittelten Daten gespeichert oder ausgedruckt? Dann melden Sie sich ab.",
+      text: "Bitte beachten Sie, dass Sie Ihren Freischaltcode nicht erneut verwenden können. Wenn Sie unseren Service nächstes Jahr wieder nutzen möchten, können Sie sich einfach erneut registrieren. Haben Sie Ihre Steuererklärung gespeichert? Dann können Sie sich abmelden.",
```

Comment:
That we do not mention the transfer ticket is fine with Nico and Nadine?

Reply:
I am just raising that because the transfer ticket is the information that allows users to track their tax declaration send to the tax offices (but it is also on the pdf, so this should be fine)

Reply:
I will ask Nico, Nadine and Carina

Reply:
It is confirmed. The text with the transfer ticket is now removed. In further taks we may add another hint to it!